.github/workflows/test-sdk.yml

```
@@ -0,0 +1,36 @@
+name: test-sdk
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test-sdk:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)